ImageViewerAnnotate.php

```diff
@@ -12,9 +12,8 @@
 
 class ImageViewerAnnotate extends \ExternalModules\AbstractExternalModule {
 
-    private $imageViewTag = "@IMAGEVIEW";
-    // private $imagePipeTag = "@IMAGEPIPE";
-    private $imagePipeTag = "@IMAGEANNOTATE";
+    private $imageViewTag = "@IMAGE-UPLOAD-VIEW";
```

Comment:
are hyphens kosher? (what have other action tags used for their separator if any?

for the view tag, how about `@IMAGE-ANNOTATE-UPLOAD` ? I think that better ties it to our module and the viewing is a side effect?

Reply:
looking at some of the default action tags, it looks like hyphens are kosher (see below)

![image](https://user-images.githubusercontent.com/22246473/146811506-f9e98432-4986-4ae6-ac59-099230091329.png)

`@IMAGE-ANNOTATE-UPLOAD` works for me!